models/labels/labels_addresses.sql

```diff
@@ -22,7 +22,7 @@
     , ref('labels_safe_ethereum')
     , ref('labels_tornado_cash')
     , ref('labels_likely_bot_labels')
-    , ref('labels_quest_participants')
+    , ref('labels_quest_participants_legacy')
```

Comment:
on friday afternoon, we went ahead and universally built all `_legacy.sql` files and applied a `tag:legacy` on each. for all the instances where you need to update downstream refs, such as here, now you will need to edit on the `_legacy.sql` file instead

[this failure](https://github.com/duneanalytics/spellbook/actions/runs/5579589482/jobs/10195347067?pr=3614#step:10:12) is telling us that we are editing the file which is waiting to be migrated. the model isn't quite ready for dunesql yet, so we don't want to apply the tag there.

Reply:
i can double check the readme to ensure that is clear

Reply:
this more than likely applies to multiple PRs you have open

Reply:
ah, we actually went ahead and already updated all the downstream refs as well. i think you can ignore that step. i will update the readme to reflect that

Reply:
so this is saying we don't use the _legacy models any more? And it will auto-figure out which model to use?

Reply:
all spark spells are now the `_legacy` spells. anything else is either migrated to dunesql or sitting idle until picked up to migrate. all changes to spark need to be on legacy files